.github/workflows/automatic_pull_request_review.yml

```diff
@@ -75,7 +76,7 @@ jobs:
           gem --version
           bundler --version
 
-      - name: Run Danger
+      - name: Pull request review by Danger
```

Comment:
I meant `Pull request review by Danger` should be at line 9, here should be `Run Danger`. `Pull request review by Danger` is something overall for all steps below 💭